packages/desktop-client/src/components/mobile/accounts/AccountTransactions.tsx

```diff
@@ -140,12 +174,36 @@ function AccountName({ account, pending, failed }) {
   );
 }
 
-function TransactionListWithPreviews({ account }) {
-  const [currentQuery, setCurrentQuery] = useState();
+function NameOnlyHeader({ accountName }: { readonly accountName: string }) {
+  return (
+    <View
+      style={{
+        flexDirection: 'row',
+      }}
+    >
+      <Text style={{ ...(styles.lineClamp(2) as CSSProperties) }}>
+        {accountName}
+      </Text>
+    </View>
+  );
+}
+
+function TransactionListWithPreviews({
+  account,
+  accountId,
```

Comment:
I think this prop is not needed since there's already the account entity which has the id

Reply:
The account entity can be undefined for name-only accounts.